### DISCUSSION TITLE
Change name dynamically.

### DISCUSSION THREAD

**Discussion by wallacesivirino** [Clooos/Bubble-Card#948] [Questions about config, custom styles and templates] [answered]
How to dynamically change a button name based on an entity status, just like it can be done with an icon or color?

  .bubble-icon {
    color: ${hass.states['binary_sensor.statuspc'].state === 'on' ? 'rgb(252, 240, 110)' : 'rgb(41, 96, 111)'} !important;
  }

**Comment by Clooos** [comment, marked as answer]
Hi! You can check in the Templates section in the GitHub documentation, this is this example:

<img width="875" alt="image" src="https://github.com/user-attachments/assets/e80c46b1-1007-458b-8674-2be679af6d05">
